docs/tutorials/embedding/OutOfSampleEmbed.ipynb

```diff
@@ -21,7 +21,7 @@
     "\n",
     "$W \\in \\textbf{R}^{m \\times n}$ is a matrix with each row being an adjacency vector, for $m$ out-of-sample vertices.\n",
     "\n",
-    "We can obtain this estimation with ASE's `transform` method.  \n",
+    "We can obtain this estimation with ASE's or LSE's `transform` method.  \n",
     "Running through the Adjacency Spectral Embedding tutorial is recommended prior to this tutorial."
```

Comment:
can you add a link to this tutorial

Reply:
There is currently not a tutorial for LSE, do you want me to add the link to the ASE tutorial?